ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml

```diff
@@ -65,3 +65,6 @@ keys:
       hardware:
         type: dict
         $ref: "eos_cli_config_gen#/$defs/flow_tracking"
+        keys:
+          record_format_ipfix_standard_timestamps_counters:
```

Comment:
There is two options inside record `format` and `rewrite`.
So i think we should change the model to - 
```
record:
      type: dict
      keys:
          format_ipfix_standard_timestamps_counters:
                 type: bool
```

Reply:
Updated the schema as per suggestion.